leaf-ui/js/loadSaveFunctions.js

```diff
@@ -236,7 +253,7 @@ function createBBElement(cell, funcsegs){
  */
 function getModelAnalysisJson(configCollection) {
 	var obj = {};
```

Comment:
This line of code seems to be useless. I commented it out and no issue arose.

Reply:
We actually need it for structural consistency because if I only download the graph without setting an object where obj,graph = graph.toJSON() then when the file is loaded because there isn't any obj.graph it can't be loaded